dor-workflow-service.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ['lib']
 
   gem.add_dependency 'activesupport', '>= 3.2.1', '< 5'
-  gem.add_dependency 'nokogiri', '~> 1.6.0'
+  gem.add_dependency 'nokogiri', '~> 1.7'
```

Comment:
Good question, but anything using nokogiri 1.6.x will fail a bundle audit, it has security vulnerabilities.  The bundle audits indicate that some 1.7.x are OK.